compiler/lib/effects.ml

```diff
@@ -355,34 +441,37 @@ let allocate_continuation ~st ~alloc_jump_closures ~split_closures pc x cont =
      closure to bind [x] if it is used in the loop body. In
      other cases, we can just pass the closure corresponding
      to the next block. *)
-  let pc', args = cont in
+  let _, args = cont in
```

Comment:
I think it would be clearer to remove the parameter `direct_pc`, and maybe rename `cont` into `direct_cont`. One would then write:
```suggestion
  let direct_pc, args = direct_cont in
```